doc/apache_setup.md

```diff
@@ -1,106 +1,18 @@
-# Worldview installation using Apache
+# How to Setup Worldview Using Apache
 
-Download and install [Node.js](http://nodejs.org/) using the instructions on
-their site.
+> Note: Using Worldview with Apache is deprecated and will not be supported in the future. Is is recommended that you use Worldview with Node/[Express](https://expressjs.com/) instead. If you choose to proceed with Apache, choose one of the setup methods below.
 
-Clone this repository:
+## RPM Setup
 
-```bash
-git clone https://github.com/nasa-gibs/worldview.git
-cd worldview
-```
+* Run `bash ./tasks/buildAll.sh` to generate an `.rpm` file in `/build/rpm`.
+* Deploy the `.rpm` file to your server.
+* Note: you may have to modify `httpd.conf` and `worldview.spec` to match your environment before running these steps.
 
-Select one of the following configuration repositories:
+## Manual Setup
 
-```bash
-# Official EOSDIS configurations
-git clone https://github.com/nasa-gibs/worldview-options-eosdis.git options
-
-# Or a blank repository with only Corrected Reflectance and no branding
-git clone https://github.com/nasa-gibs/worldview-options-template.git options
-```
-
-Now install a global version of [grunt](http://gruntjs.com/) using the
-following command:
-
-```bash
-sudo npm install --global grunt-cli
-```
-
-Install the packages required for building:
-
-```bash
-npm install
-```
-
-Some python libraries are required. Install virtualenv to keep additional
-libraries installed in a local directory:
-
-```bash
-sudo easy_install virtualenv==1.10.1
-```
-
-Start the build with:
-
-```bash
-grunt
-```
-
-The ``dist`` directory now contains ``site`` tar files that can be
-can be uploaded and unpacked to a web server.
-
-## Mac OS X
-### Apache Server Configuration
-*These instructions work with OS X El Capitan. Directories and steps may differ with other OS X versions*
-
-Open httpd.conf
-```bash
-sudo vi /etc/apache2/httpd.conf
-```
-Make sure the following modules are enabled:
-```bash
-LoadModule expires_module libexec/mod_expires.so
```

Comment:
I think this is important information to retain. Without some documentation about the apache modules needed locally, it would be hard for a user to discern why their apache instance is not properly serving WV.

Reply:
My two cents is that we shouldn't officially support Apache. We're migrating to Node/Express officially. Worldview will run on any web server, Apache, Express, etc. Pat would like to remove CGI dependencies, and I agree, which would mean we could even host Worldview in a static environment like S3 or the NGAP equivalent. I think providing support for Apache or any other environment than what we'll be using in development and production means spending time and resources that could be better spent on new features. Anyone who insists on using Apache is likely familiar with how to configure an Apache server to serve static assets and execute CGI files. It's a fairly standard Apache configuration and I think we should leave the documentation for how to do it to the Apache community. That being said, I'm flexible on this. One possibility is having a link on this page to Apache docs for how to configure a web app. We can discuss more on Monday.

Reply:
I agree that Apache is not something we should focus on. With that being said, Worldview can run Apache currently and it is a configuration that could be in use in some local environments. 

I would argue your point that "anyone who insists on using Apache is likely familiar with how to configure an Apache server to serve static assets and execute CGI files". LAMP stack applications like WAMP, XAMPP, etc. make it extremely easy to get an Apache environment setup locally without having much familiarity with how Apache works. I think it would be a disservice to our users to remove this information as of now until we fully remove support for Apache.

Agree, let's discuss more the future of supporting Apache and come up with a game plan to either keep supporting it our gracefully remove it.